packages/react-reconciler/src/__tests__/useEvent-test.internal.js

```diff
@@ -0,0 +1,595 @@
+/**
```

Comment:
Remove the `.internal` from the file name. That's an old opt-out we added when we started testing the actual npm release artifacts in CI, as opposed to just the source files.